index.js

```diff
@@ -1,3 +1,4 @@
+// @ts-expect-error Typing of css-tree is incomplete
```

Comment:
Props for expect error with description instead of ignore

Reply:
Learned it from the best! 😉